app/labor/markdown_parser.rb

```diff
@@ -108,7 +108,7 @@ def prefix_all_images(html, width = 880)
       next if allowed_image_host?(src)
 
       img["loading"] = "lazy"
-      img["src"] = if giphy_img?(src)
+      img["src"] = if GiphyService.new.giphy_img?(src)
```

Comment:
What do you think about a clearer name like `Giphy::Image.valid_url?` or `Giphy::Image.valid_source?`

Reply:
That looks a lot cleaner, great suggestion.